app/js/tag.js

```diff
@@ -29,11 +29,17 @@
   }
   setTimeout(function () {
     if (!global.UDTracker) {
-      console.warn('Blocked USERDIVE Scripts') // eslint-disable-line no-console
+      element.setAttribute(
+        '${stateName}',
+        JSON.stringify({ warning: 'Blocked USERDIVE Scripts' })
+      )
       return
     }
     if (!global.UDTracker.cookie.enableSession()) {
-      console.warn('Failed start USERDIVE') // eslint-disable-line no-console
+      element.setAttribute(
+        '${stateName}',
+        JSON.stringify({ warning: 'Failed start USERDIVE' })
```

Comment:
Because we want to implement client only for state management, please make error message on extension side